a10_octavia/controller/worker/flows/a10_load_balancer_flows.py

```diff
@@ -293,6 +290,10 @@ def get_new_lb_networking_subflow(self, topology):
             new_LB_net_subflow.add(vthunder_tasks.EnableInterface(
                 name="backup_enable_interface",
                 rebind=[a10constants.BACKUP_VTHUNDER]))
+            new_LB_net_subflow.add(a10_database_tasks.MarkVThunderStatusInDB(
+                name="Set_backup_vThunder_entry_active",
```

Comment:
Names should be a constant and formatted as follows: https://github.com/openstack/octavia/blob/master/octavia/common/constants.py#L422